general_rules/ExternalDevices.tex

```diff
@@ -59,10 +59,16 @@ \subsubsection{Technical Specifications}
   \item \textbf{Storage:} Unrestricted.
 \end{itemize}
 
-No other brands or models will be accepted. There are no constrains regarding the software installed in the OSL but no additional hardware is allowed.
+No other brands or models will be accepted. There are no constrains regarding the software installed in the \iterm{OSL} but no additional hardware is allowed.
 
-The referees, Technical Committee, and Organizing Committee members may run random checks anytime during the competition prior to the test to verify that the laptop in the TOYOTA HSR \iterm{Mounting Bracket} has no additional hardware plugged in, and matches the authorized specifications.
+The referees, Technical Committee, and Organizing Committee members may run random checks anytime during the competition prior to the test to verify that the laptop in the Toyota HSR \iterm{Mounting Bracket} has no additional hardware plugged in, and matches the authorized specifications.
 
+\subsubsection{External Wi-Fi adapter}
+As described in section 6.2.2.5.6 of the Toyota HSR manual, an external Wi-Fi adapter can be used to increase stability and/or bandwidth of the network connection. As teams are allowed to use the network adapter of the \iterm{OSL}, teams may also chose to use an external Wi-Fi adapter.
+
+No constraints are applied regards the Wi-Fi specifications. However to be used on the Toyota HSR, the Wi-Fi adapter should be USB-powered. To allow wired network connections between the internal computer, the \iterm{OSL} and the external Wi-Fi adapter, a (USB-powered) switch is allowed.
```

Comment:
"in regards to the Wi-Fi" or "regarding the Wi-Fi"
And it should be "adapter specifications.", right?
Is there enough space in the backback for the standard laptop and a switch?
Or more general do we still require all this additional equipment to fit inside the backback?

Reply:
Fix the grammar issue.

A switch and Wi-Fi adapter don't fit in the backpack. We taped them on side of the robot. As the base is bigger than the torso. These devices don't or just barely increase the size of the robot.